python/hsfs/engine/python.py

```diff
@@ -121,12 +125,24 @@ def _sql_offline(
         hive_config=None,
     ):
         if arrow_flight_client.get_instance().is_flyingduck_query_object(sql_query):
-            result_df = arrow_flight_client.get_instance().read_query(sql_query)
+            result_df = util.run_with_loading_animation(
+                "Reading data from Hopsworks, using FlyingDuck",
+                arrow_flight_client.get_instance().read_query,
+                sql_query,
+            )
         else:
             with self._create_hive_connection(
                 feature_store, hive_config=hive_config
             ) as hive_conn:
-                result_df = pd.read_sql(sql_query, hive_conn)
+                # Suppress SQLAlchemy pandas warning
+                with warnings.catch_warnings():
```

Comment:
Can we please not do this? I usually keep an eye on warnings for things that will be deprecated soon.

Reply:
It's about this warning in particular: "UserWarning: pandas only support SQLAlchemy connectable(engine/connection) ordatabase string URI or sqlite3 DBAPI2 connectionother DBAPI2 objects are not tested, please consider using SQLAlchemy", we should get rid of it.

Deprecation warning ragarding the connection will still show up, like this one here, that's why I didn't include it in the block.
![image](https://github.com/logicalclocks/feature-store-api/assets/4321121/6d9eec0c-d981-4bab-a53d-ca81ba2e3123)

Reply:
we can specific the filter to only filter user warning `warnings.simplefilter("ignore", UserWarning),` then deprecation warning will still show. I'll make a commit.